libs/styles/variables.css

```diff
@@ -30,6 +30,7 @@
 
   /* Fonts */
   --body-font-family: 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif;
+  --code-font-family: 'Source Code Pro', source-code-pro, monospace;
   --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
```

Comment:
I checked all repos under org and nothing is using `--fixed-font-family`. Delete?

Reply:
HelpX seems to use it in their [styles](https://github.com/adobecom/helpx/blob/37b7eb6956405970321e53657b140e32c6784a1a/styles/styles.css#L108), but they also define it [themselves](https://github.com/adobecom/helpx/blob/37b7eb6956405970321e53657b140e32c6784a1a/styles/styles.css#L24), so this does seem redundant to me.